### ISSUE TITLE
where is the softmax?

### ISSUE THREAD

**Issue by xhjcxxl** [yubowen-ph/JointER#7]
in the code, i can't see the softmax in the HE extraction to extracte subject head tag. there is a FC network which named nn.linear()?
can you tell me where is the softmax?

**Comment by yubowen-ph** [comment]
Hi,

Thanks for your interest in this work. The outputs of the FC layer are denoted as the logits and passed to the CrossEntropyLoss function. In Pytorch, Softmax has been implemented in nn.CrossEntropyLoss. At inferentence time, argmax (logits) is equivalent to argmax (Softmax(logits)) since Softmax does not change the order of values in the logits.

Best regards,
Yu Bowen
